src/libraries/Microsoft.Extensions.Logging.Configuration/src/LoggerFilterConfigureOptions.cs

```diff
@@ -50,6 +51,10 @@ private void LoadDefaultConfigValues(LoggerFilterOptions options)
                     }
                 }
             }
+
+            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+                Justification = "IConfiguration.GetValue<T> is safe when T is not generic. In this case, T is bool.")]
```

Comment:
This message could be cleaned up to just:

"IConfiguration.GetValue<T> is safe when T is a bool."